test/unit/behaviors.spec.js

```diff
@@ -16,390 +24,480 @@ describe('Behaviors', function() {
   });
 
   describe('behavior parsing with a functional behavior lookup', function() {
```

Comment:
We can move this test from here into `config` folder, just for testing `behaviorsLookup` function. Thoughts?